src/lava/lib/optimization/solvers/generic/solver.py

```diff
@@ -207,6 +258,45 @@ def _get_requirements_and_protocol(self,
         else:
             raise NotImplementedError(str(backend) + backend_msg)
 
+    def _get_run_config(self, backend):
+        if backend in CPUS:
+            pdict = {self.solver_process: self.solver_model,
+                     ReadGate: ReadGatePyModel,
+                     Dense: PyDenseModelFloat,
+                     StochasticIntegrateAndFire: StochasticIntegrateAndFireModel
+                     }
+            run_cfg = Loihi1SimCfg(exception_proc_model_map=pdict,
+                                   select_sub_proc_model=True)
+        elif backend in NEUROCORES:
+            pdict = {self.solver_process: self.solver_model,
+                     ReadGate: ReadGateCModel,

```

Comment:
If you really need to define these ProcModels in a public repo without access to collab; maybe import them here only elif backend in NEUROCORES?